CIP-0068/README.md

```diff
@@ -204,17 +212,19 @@ metadata =
     description : bounded_bytes, ; UTF-8
     ? ticker: bounded_bytes, ; UTF-8
     ? url: bounded_bytes, ; UTF-8
-    ? logo: uri,
     ? decimals: int
+
+    ; 'logo' does not follow the explanation of the token-registry, it needs to be a valid URI and not a plain bytestring.
+    ; The logo URI must point to a resource with media type (mime type) `image/png`, `image/jpeg` or `image/svg+xml`.
+    ? logo: uri,
+
     ; ... Additional properties are allowed
   }
 
-; A URI as a UTF-8 encoded bytestring.
-; The URI scheme must be one of `https`, `ipfs` or `data`
-; Do not encode plain file payloads as URI.
-; 'logo' does not follow the explanation of the token-registry, it needs to be a valid URI and not a plain bytestring.
-; Only use the following media types: `image/png`, `image/jpeg`, `image/svg+xml`
-uri = bounded_bytes
+; A valid Uniform Resource Identifier (URI) as a UTF-8 encoded bytestring.
+; The URI scheme must be one of `https` (HTTP), `ipfs` (IPFS), `ar` (Arweave) or `data` (on-chain).
+; Data URLs (on-chain data) must comply to RFC2397.
+uri = bounded_bytes ; UTF-8
```

Comment:
Point of clarification cause I may have misunderstood here, the token registry stores the image in *ONLY* data: format by default and that is actually an acceptable URI in this case but we just want it to be a "plain text" URI rather than a raw bytestring?

Reply:
I think this part was written by @AndrewWestberg, maybe he can clarify as I just moved things around.
In the token registry, images are base64 encoded PNGs, they are not data URLs, but I think data urls are valid here.

Reply:
@alessandrokonrad can comment on this. My addition was only to add 444 which is a copy paste of parts of 222 and 333.